addons/website_event/views/event_templates_page.xml

```diff
@@ -47,31 +47,17 @@
         <div id="o_wevent_submenu_desktop" class="d-none d-lg-block mt-3 mb-3 mb-lg-2">
```

Comment:
For every change on `o_wevent_submenu_desktop`, there's an equal and opposite change in `o_wevent_submenu_mobile`.

Seems the breadcrumbs are still presend on mobile ^^